lib/grape-swagger/doc_methods.rb

```diff
@@ -491,7 +497,7 @@ def setup(options)
         end
 
         ops = visible_ops.group_by do |route|
-          @@documentation_class.parse_path(route.route_path, api_version)
+          @@documentation_class.parse_path(route.route_path, api_version, hide_module_from_path)
```

Comment:
That is a definite problem and feel free to fix it up in a different PR. Want to do that first and we'll merge this after?

My biggest gripe is introducing N parameters here.

I think this like should be:

```ruby
documentation_class.parse_path(options.merge(path: route.route_path, version: api_version))
```